### ISSUE TITLE
Grid2viz and Grid2Op=0.7.1 error: ModuleNotFoundError: No module named 'grid2op.Observation.CompleteObservation'; 'grid2op.Observation' is not a package

### ISSUE THREAD

**Issue by KishanGitASU** [mjothy/grid2viz#14]
Hi,

I was able install grid2viz using the instructions on the github page. I was able to run the built-in data from grid2viz using `launch_grid2viz.py`. I was also able to run grid2viz on the scenarios generated by my user defined agent from grid2op (when grid2op=0.5.8). However, when I updated to grid2op =0.7.1 (using make_new to create environment), the scenarios generated by the grid2op runner are not able to be processed by grid2viz.

_I am not sure if it is an error from my end or there is some issue_. I am attaching the scenario that I generated below. Please let me know if it is because of grid2op=0.7.1 or you are able to visualize the attached data using grid2viz.

**Data**: [Link to download the scenario data generated by grid2op=0.7.1 & make_new function](https://drive.google.com/drive/folders/1GMDApiA00yTzVw7qhi-E_AHduE1whjWx?usp=sharing)

**More information**: Grid2op =0.7.1, my agent uses the environment generated by make_new. The synatx used is : `env = make_new("l2rpn_case14_sandbox",reward_class=CombinedReward)` 

```
Chronics location that being used: C:\Users\kisha\data_grid2op\l2rpn_case14_sandbox\chronics
Grid location being used: C:\Users\kisha\data_grid2op\l2rpn_case14_sandbox\grid.json
Reward class that is being used: <class 'grid2op.Reward.CombinedReward.CombinedReward'>
Action type class being used: <class 'grid2op.Action.TopologyAndDispatchAction.TopologyAndDispatchAction'>
Observation type class being used: <class 'grid2op.Observation.CompleteObservation.CompleteObservation'>
Legal action class being used: <class 'grid2op.Rules.DefaultRules.DefaultRules'>
Voltage controller class being used: <class 'grid2op.VoltageControler.ControlVoltageFromFile.ControlVoltageFromFile'>
```

**Full trace of error**: 
```
(grid2viz) C:\Users\kisha\Grid2Viz>py C:\Users\kisha\Grid2Viz\grid2viz\main.py --path D:\baseline_me\agents_log
 * Serving Flask app "C:\Users\kisha\Grid2Viz\grid2viz\index.py"
 * Environment: production
   WARNING: This is a development server. Do not use it in a production deployment.
   Use a production WSGI server instead.
 * Debug mode: off
c:\users\kisha\miniconda3\envs\grid2viz\lib\site-packages\pandapower\io_utils.py:8: FutureWarning:

pandas.util.testing is deprecated. Use the functions in the public API at pandas.testing instead.

the config file used is located at: C:\Users\kisha\Grid2Viz\config.ini
Agents ata used are located at: D:\baseline_me\agents_log
Data used are located at: D:\baseline_me\agents_log
 * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
127.0.0.1 - - [29/Apr/2020 23:48:19] "[37mGET / HTTP/1.1[0m" 200 -
127.0.0.1 - - [29/Apr/2020 23:48:20] "[37mGET /_dash-layout HTTP/1.1[0m" 200 -
127.0.0.1 - - [29/Apr/2020 23:48:20] "[37mGET /_dash-dependencies HTTP/1.1[0m" 200 -
127.0.0.1 - - [29/Apr/2020 23:48:20] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
127.0.0.1 - - [29/Apr/2020 23:48:20] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
127.0.0.1 - - [29/Apr/2020 23:48:20] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
127.0.0.1 - - [29/Apr/2020 23:48:20] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
127.0.0.1 - - [29/Apr/2020 23:48:20] "[37mPOST /_dash-update-component HTTP/1.1[0m" 200 -
Exception on /_dash-update-component [POST]
Traceback (most recent call last):
  File "c:\users\kisha\miniconda3\envs\grid2viz\lib\site-packages\flask\app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\users\kisha\miniconda3\envs\grid2viz\lib\site-packages\flask\app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "c:\users\kisha\miniconda3\envs\grid2viz\lib\site-packages\flask\app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "c:\users\kisha\miniconda3\envs\grid2viz\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "c:\users\kisha\miniconda3\envs\grid2viz\lib\site-packages\flask\app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "c:\users\kisha\miniconda3\envs\grid2viz\lib\site-packages\flask\app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "c:\users\kisha\miniconda3\envs\grid2viz\lib\site-packages\dash\dash.py", line 1459, in dispatch
    response.set_data(self.callback_map[output]["callback"](*args))
  File "c:\users\kisha\miniconda3\envs\grid2viz\lib\site-packages\dash\dash.py", line 1339, in add_context
    output_value = func(*args, **kwargs)  # %% callback invoked %%
  File "C:\Users\kisha\Grid2Viz\grid2viz\src\episodes\episodes_clbk.py", line 32, in load_scenario_cards
    best_agent_episode = make_episode(best_agents[scenario]['agent'], scenario)
  File "C:\Users\kisha\Grid2Viz\grid2viz\src\manager.py", line 49, in make_episode
    episode = compute_episode(episode_name, agent)
  File "C:\Users\kisha\Grid2Viz\grid2viz\src\manager.py", line 89, in compute_episode
    path, episode_name
  File "c:\users\kisha\miniconda3\envs\grid2viz\lib\site-packages\grid2op\EpisodeData.py", line 176, in from_disk
    os.path.join(agent_path, EpisodeData.OBS_SPACE))
  File "c:\users\kisha\miniconda3\envs\grid2viz\lib\site-packages\grid2op\Observation.py", line 1369, in from_dict
    tmp = SerializableSpace.from_dict(dict_)
  File "c:\users\kisha\miniconda3\envs\grid2viz\lib\site-packages\grid2op\Space.py", line 1494, in from_dict
    exec("from {} import {}".format(".".join(actionClass_li[:-1]), actionClass_li[-1]))
  File "<string>", line 1, in <module>
ModuleNotFoundError: No module named 'grid2op.Observation.CompleteObservation'; 'grid2op.Observation' is not a package
127.0.0.1 - - [29/Apr/2020 23:48:21] "[35m[1mPOST /_dash-update-component HTTP/1.1[0m" 500 -
You pressed Ctrl+C! Shutting down Grid2Viz!
```

**Comment by mjothy** [comment]
Hello @KishanGitASU ,

Your problem seems related to Pull Request #13 which will upgrade Grid2Viz to the newer version of Grid2op.
May I ask you to try again once the pull request is accepted and merged ? It should be done during the week.

Thank you for your report !

Mario

**Comment by mjothy** [comment]
Hello,

Your issue should be solved with the last PR. If you can try again, I can close this issue if the problem is solved for you.

Mario

**Comment by KishanGitASU** [comment]
Hi Mario,

I apologize for the delay. It seems my virtual environment and path variables somehow got messed up. I had to re-install everything which lead to more errors and conflicts since I am using some very old packages in my virtual environment.

Finally, I checked the Grid2Viz with the latest version of Grid2Op and it is working great! :) 

I had a small observation, I got an error saying "No module named dill found" when I ran the Grid2Viz.  I just installed it and then it started to worked. I am mentioning this just in case if it the above error is not expected to occur. Thank you :)
